# This file is distributed under the same license as the Django package.
# 
msgid ""
msgstr ""
"Project-Id-Version: Django\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2020-06-26 17:01+0300\n"
"PO-Revision-Date: 2010-05-13 15:35+0200\n"
"Last-Translator: Django team\n"
"Language-Team: English <en@li.org>\n"
"Language: en\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

#: .\sanor\Lib\site-packages\django\contrib\gis\apps.py:8
msgid "GIS"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\db\models\fields.py:64
msgid "The base GIS field."
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\db\models\fields.py:201
msgid ""
"The base Geometry field -- maps to the OpenGIS Specification Geometry type."
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\db\models\fields.py:283
msgid "Point"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\db\models\fields.py:290
msgid "Line string"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\db\models\fields.py:297
msgid "Polygon"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\db\models\fields.py:304
msgid "Multi-point"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\db\models\fields.py:311
msgid "Multi-line string"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\db\models\fields.py:318
msgid "Multi polygon"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\db\models\fields.py:325
msgid "Geometry collection"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\db\models\fields.py:331
msgid "Extent Aggregate Field"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\db\models\fields.py:346
msgid "Raster Field"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\forms\fields.py:19
msgid "No geometry value provided."
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\forms\fields.py:20
msgid "Invalid geometry value."
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\forms\fields.py:21
msgid "Invalid geometry type."
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\forms\fields.py:22
msgid ""
"An error occurred when transforming the geometry to the SRID of the geometry "
"form field."
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\templates\gis\admin\openlayers.html:35
#: .\sanor\Lib\site-packages\django\contrib\gis\templates\gis\openlayers.html:12
msgid "Delete all Features"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\templates\gis\admin\openlayers.html:37
msgid "WKT debugging window:"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\templates\gis\openlayers.html:13
msgid "Debugging window (serialized value)"
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\views.py:8
msgid "No feeds are registered."
msgstr ""

#: .\sanor\Lib\site-packages\django\contrib\gis\views.py:14
#, python-format
msgid "Slug %r isn't registered."
msgstr ""
